cypress/integration/login.spec.js

```diff
@@ -44,13 +44,10 @@ describe('login', function() {
 
     cy.wait('@postLogin')
       .its('status')
-      .should('eq', 401);
+      .should('eq', 400);
 
     cy.url().should('contain', '/login');
-    cy.get('div[role="alert"]').should(
-      'have.text',
-      'The email or password you entered is incorrect!',
-    );
+    cy.get('div[role="alert"]').should('have.text', 'Unable to log in with provided credentials.');
```

Comment:
Glad you enabled global error handling, but can we still tune error messages on specific status codes / errors? Isn’t prefer the old error message here.